Fix failing IERS tests

The CI testing is failing due to IERS inaccuracies.

Errors:

```
    def test_create_mount_simulator(config_host, config_port):
        # Use the simulator create function directly.
>       mount = create_mount_simulator()

tests/test_mount.py:27: 
```

and

```
E               astropy.utils.iers.iers.IERSRangeError: (some) times are outside of range covered by IERS table. Cannot convert with full accuracy. To allow conversion with degraded accuracy set astropy.utils.iers.conf.iers_degraded_accuracy to "warn" or "silent". For more information about setting this configuration parameter or controlling its value globally, see the Astropy configuration system documentation https://docs.astropy.org/en/stable/config/index.html.
```

--- a/src/panoptes/pocs/mount/mount.py
+++ b/src/panoptes/pocs/mount/mount.py
@@ -216,7 +216,7 @@ class AbstractMount(PanBase):
         self._tracking_rate = value
 
     def set_park_coordinates(self, ha=-170 * u.degree, dec=-10 * u.degree):
-        """ Calculates the RA-Dec for the the park position.
+        """ Calculates the RA-Dec for the park position.
 
         This method returns a location that points the optics of the unit down toward the ground.
 

--- a/src/panoptes/pocs/mount/mount.py
+++ b/src/panoptes/pocs/mount/mount.py
@@ -215,46 +215,6 @@ class AbstractMount(PanBase):
         """ Set the tracking rate """
         self._tracking_rate = value
 
-    def set_park_coordinates(self, ha=-170 * u.degree, dec=-10 * u.degree):
-        """ Calculates the RA-Dec for the park position.
-
-        This method returns a location that points the optics of the unit down toward the ground.
-
-        The RA is calculated from subtracting the desired hourangle from the
-        local sidereal time. This requires a proper location be set.
-
-        Note:
-            Mounts usually don't like to track or slew below the horizon so this
-                will most likely require a configuration item be set on the mount
-                itself.
-
-        Args:
-            ha (Optional[astropy.units.degree]): Hourangle of desired parking
-                position. Defaults to -165 degrees.
-            dec (Optional[astropy.units.degree]): Declination of desired parking
-                position. Defaults to -165 degrees.
-
-        Returns:
-            park_skycoord (astropy.coordinates.SkyCoord): A SkyCoord object
-                representing current parking position.
-        """
-        self.logger.debug('Setting park position')
-
-        park_time = current_time()
-        park_time.location = self.location
-
-        lst = park_time.sidereal_time('apparent')
-        self.logger.debug(f'LST: {lst}')
-        self.logger.debug(f'HA: {ha}')
-
-        ra = lst - ha
-        self.logger.debug(f'RA: {ra}')
-        self.logger.debug(f'Dec: {dec}')
-
-        self._park_coordinates = SkyCoord(ra, dec)
-
-        self.logger.debug(f'Park Coordinates RA-Dec: {self._park_coordinates}')
-
     def get_target_coordinates(self):
         """ Gets the RA and Dec for the mount's current target. This does NOT necessarily
         reflect the current position of the mount, see `get_current_coordinates`.
@@ -606,16 +566,17 @@ class AbstractMount(PanBase):
     def park(self, *args, **kwargs):
         """ Slews to the park position and parks the mount.
 
+        The park position must be set manually first for this method to work.
+
+        Most mount subclasses will override this method to provide mount-specific
+        park functionality.
+
         Note:
             When mount is parked no movement commands will be accepted.
 
         Returns:
             bool: indicating success
         """
-
-        self.set_park_coordinates()
-        self.set_target_coordinates(self._park_coordinates)
-
         response = self.query('park')
 
         if response:

--- a/src/panoptes/pocs/mount/simulator.py
+++ b/src/panoptes/pocs/mount/simulator.py
@@ -151,6 +151,11 @@ class Mount(AbstractMount):
 
         self.stop_slew(next_position='is_home')
 
+    def set_park_coordinates(self, ha=-170 * u.degree, dec=-10 * u.degree):
+        """ Set the park coordinates for the mount simulator """
+        self.logger.debug('Setting park coordinates')
+        self._park_coordinates = self._mount_coord_to_skycoord([ha, dec])
+
     def park(self):
         """ Sets the mount to park for simulator """
         self.logger.debug('Setting to park')

--- a/src/panoptes/pocs/mount/simulator.py
+++ b/src/panoptes/pocs/mount/simulator.py
@@ -19,9 +19,6 @@ class Mount(AbstractMount):
 
         self._loop_delay = self.get_config('loop_delay', default=0.01)
 
-        self.set_park_coordinates()
-        self._current_coordinates = self._park_coordinates
-
         self.logger.debug('Simulator mount created')
 
     def initialize(self, unpark=False, *arg, **kwargs):
@@ -150,11 +147,6 @@ class Mount(AbstractMount):
         self._is_parked = False
 
         self.stop_slew(next_position='is_home')
-
-    def set_park_coordinates(self, ha=-170 * u.degree, dec=-10 * u.degree):
-        """ Set the park coordinates for the mount simulator """
-        self.logger.debug('Setting park coordinates')
-        self._park_coordinates = self._mount_coord_to_skycoord([ha, dec])
 
     def park(self):
         """ Sets the mount to park for simulator """

--- a/tests/bisque/test_mount.py
+++ b/tests/bisque/test_mount.py
@@ -56,22 +56,6 @@ def test_initialize(mount):
     assert mount.initialize() is True
 
 
-def test_set_park_coords(mount):
-    mount.initialize()
-    assert mount._park_coordinates is None
-
-    mount.set_park_coordinates()
-    assert mount._park_coordinates is not None
-
-    assert mount._park_coordinates.dec.value == -10.0
-    assert mount._park_coordinates.ra.value - 322.98 <= 1.0
-
-    mount.set_park_coordinates()
-
-    assert mount._park_coordinates.dec.value == -10.0
-    assert mount._park_coordinates.ra.value - 172.57 <= 1.0
-
-
 def test_unpark_park(mount):
     assert mount.is_parked is True
     mount.initialize()

--- a/tests/test_ioptron.py
+++ b/tests/test_ioptron.py
@@ -57,17 +57,6 @@ class TestMount(object):
     def test_version(self):
         assert self.mount.query('version') == 'V1.00'
 
-    def test_set_park_coords(self):
-        self.mount.initialize()
-        assert self.mount._park_coordinates is None
-
-        self.mount.set_park_coordinates()
-        assert self.mount._park_coordinates is not None
-
-        # These are the empirically determined coordinates for PAN001
-        assert self.mount._park_coordinates.dec.value == -10.0
-        assert self.mount._park_coordinates.ra.value - 322.98 <= 1.0
-
     def test_unpark_park(self):
         assert self.mount.is_parked is True
         self.mount.initialize()

--- a/tests/test_mount_simulator.py
+++ b/tests/test_mount_simulator.py
@@ -53,21 +53,6 @@ def test_target_coords(mount):
     assert mount.get_target_coordinates().to_string() == '300.182 22.7109'
 
 
-def test_set_park_coords(mount):
-    os.environ['POCSTIME'] = '2016-08-13 23:03:01'
-    mount.set_park_coordinates()
-    assert mount._park_coordinates is not None
-
-    assert mount._park_coordinates.dec.value == -10.0
-    assert mount._park_coordinates.ra.value - 322.98 <= 1.0
-
-    os.environ['POCSTIME'] = '2016-08-13 13:03:01'
-    mount.set_park_coordinates()
-
-    assert mount._park_coordinates.dec.value == -10.0
-    assert mount._park_coordinates.ra.value - 172.57 <= 1.0
-
-
 def test_status(mount):
     status1 = mount.status
     assert 'mount_target_ra' not in status1

--- a/tests/utils/test_logger.py
+++ b/tests/utils/test_logger.py
@@ -16,7 +16,7 @@ def test_base_logger(caplog, profile, tmp_path):
     time.sleep(1)  # Wait for log to make it there.
 
     see_log = False
-    for rec in caplog.records[-5:]:
+    for rec in caplog.records:
         if rec.message == 'Hello':
             see_log = True
 

